src/Query/Builder.php

```diff
@@ -122,7 +129,8 @@ public function search(string $query, bool $documentsAsArray = false): SearchRes
                 $this->numericFilters,
                 explode(' ', array_reduce($this->geoFilters, function ($previous, $next) {
                     return $previous . $next;
-                }))
+                })),
+                $this->sortBy
```

Comment:
After you make $this->sortBy a string, you probably need to explode the string into a list of arguments here: explode(' ', $this->sortBy)

Reply:
Fixed